File: SustitucionMOA/src/app/common/models/ordenes-de-carga-fason/crearOrdenDeCargaFasonRequest.ts
import { UnidadTransporteCarga } from "../ordenes-de-carga-common/unidadTransporteCarga";
import { OrdenDeCargaFasonDto } from "./ordenDeCargaFasonDto";

export class CrearOrdenDeCargaFasonRequest extends OrdenDeCargaFasonDto {
    
    public UnidadesTransporte: UnidadTransporteCarga[] = [];
}

File: SustitucionMOAModel/Dto/OrdenDeCargaFason/CrearOrdenDeCargaFasonRequest.cs

using SustitucionMOAModel.Dto.OrdenDeCargaCommon;
using System.Collections.Generic;

namespace SustitucionMOAModel.Dto.OrdenDeCargaFason
{
    public class CrearOrdenDeCargaFasonRequest : OrdenDeCargaFasonRequest
    {
        public List<UnidadTransporteCarga> UnidadesTransporte { get; set; }
    }
}
